cmd/machine-config-daemon/start.go

```diff
@@ -154,6 +154,12 @@ func runStartCmd(cmd *cobra.Command, args []string) {
 	}
 
 	if startOpts.onceFrom == "" {
+		status, err := dn.NodeUpdaterClient.GetStatus()
+		if err != nil {
+			glog.Fatalf("unable to get status: %s", err)
```

Comment:
Minor: "unable to get *rpm-ostree* status" ?

Reply:
Sure, done.